diff -u -r wireshark-1.4.0.orig/epan/libwireshark.def wireshark-1.4.0/epan/libwireshark.def
--- wireshark-1.4.0.orig/epan/libwireshark.def	2010-08-29 15:17:32.000000000 -0700
+++ wireshark-1.4.0/epan/libwireshark.def	2010-09-28 21:36:46.000000000 -0700
@@ -121,6 +121,8 @@
 create_dissector_handle
 create_persconffile_dir
 create_persconffile_profile
+crypt_rc4
+crypt_rc4_init
 data_out_file                   DATA
 dcerpc_add_conv_to_bind_table
 dcerpc_get_proto_hf_opnum
@@ -913,6 +915,7 @@
 show_exception
 show_fragment_seq_tree
 show_fragment_tree
+show_reported_bounds_error
 sid_name_snooping               DATA
 sid_name_table                  DATA
 smb2_cmd_vals                    DATA
diff -u -r wireshark-1.4.0.orig/plugins/Makefile.nmake wireshark-1.4.0/plugins/Makefile.nmake
--- wireshark-1.4.0.orig/plugins/Makefile.nmake	2010-08-29 15:16:09.000000000 -0700
+++ wireshark-1.4.0/plugins/Makefile.nmake	2011-01-30 14:06:12.000000000 -0800
@@ -21,6 +21,7 @@
 	sercosiii   \
 	stats_tree  \
 	unistim     \
+	uru         \
 	wimax       \
 	wimaxasncp
 
diff -u -r wireshark-1.4.0.orig/packaging/nsis/wireshark.nsi wireshark-1.4.0/packaging/nsis/wireshark.nsi
--- wireshark-1.4.0.orig/packaging/nsis/wireshark.nsi	2010-08-29 15:17:43.000000000 -0700
+++ wireshark-1.4.0/packaging/nsis/wireshark.nsi	2010-09-28 21:36:46.000000000 -0700
@@ -890,6 +890,7 @@
 File "..\..\plugins\profinet\profinet.dll"
 File "..\..\plugins\sercosiii\sercosiii.dll"
 File "..\..\plugins\unistim\unistim.dll"
+File "..\..\plugins\uru\uru.dll"
 File "..\..\plugins\wimax\wimax.dll"
 File "..\..\plugins\wimaxasncp\wimaxasncp.dll"
 !include "custom_plugins.txt"
diff -u -r wireshark-1.4.0.orig/Makefile.am wireshark-1.4.0/Makefile.am
--- wireshark-1.4.0.orig/Makefile.am	2010-08-29 15:17:53.000000000 -0700
+++ wireshark-1.4.0/Makefile.am	2010-09-28 21:36:46.000000000 -0700
@@ -279,6 +279,7 @@
 	-dlopen plugins/sercosiii/sercosiii.la \
 	-dlopen plugins/stats_tree/stats_tree.la \
 	-dlopen plugins/unistim/unistim.la \
+	-dlopen plugins/uru/uru.la \
 	-dlopen plugins/wimax/wimax.la
 
 WTAP_PLUGIN_SOURCES = \
diff -u -r wireshark-1.4.0.orig/configure.in wireshark-1.4.0/configure.in
--- wireshark-1.4.0.orig/configure.in	2010-08-29 15:17:51.000000000 -0700
+++ wireshark-1.4.0/configure.in	2010-09-28 21:36:46.000000000 -0700
@@ -1887,6 +1887,7 @@
   plugins/sercosiii/Makefile
   plugins/stats_tree/Makefile
   plugins/unistim/Makefile
+  plugins/uru/Makefile
   plugins/wimax/Makefile
   plugins/wimaxasncp/Makefile
   tools/Makefile
diff -u -r wireshark-1.4.0.orig/plugins/Makefile.am wireshark-1.4.0/plugins/Makefile.am
--- wireshark-1.4.0.orig/plugins/Makefile.am	2010-08-29 15:16:09.000000000 -0700
+++ wireshark-1.4.0/plugins/Makefile.am	2010-09-28 21:36:46.000000000 -0700
@@ -37,6 +37,7 @@
 	sercosiii \
 	stats_tree \
 	unistim \
+	uru \
 	wimax \
 	wimaxasncp
 
