Index: epan/libwireshark.def
===================================================================
--- epan/libwireshark.def	(revision 32637)
+++ epan/libwireshark.def	(working copy)
@@ -121,6 +121,8 @@
 create_dissector_handle
 create_persconffile_dir
 create_persconffile_profile
+crypt_rc4
+crypt_rc4_init
 data_out_file                   DATA
 dcerpc_add_conv_to_bind_table
 dcerpc_get_proto_hf_opnum
@@ -904,6 +906,7 @@
 show_exception
 show_fragment_seq_tree
 show_fragment_tree
+show_reported_bounds_error
 sid_name_snooping               DATA
 sid_name_table                  DATA
 smb2_cmd_vals                    DATA
Index: plugins/Makefile.nmake
===================================================================
--- plugins/Makefile.nmake	(revision 32637)
+++ plugins/Makefile.nmake	(working copy)
@@ -21,6 +21,7 @@
 	sercosiii   \
 	stats_tree  \
 	unistim     \
+	uru         \
 	wimax       \
 	wimaxasncp
 
Index: packaging/nsis/wireshark.nsi
===================================================================
--- packaging/nsis/wireshark.nsi	(revision 32637)
+++ packaging/nsis/wireshark.nsi	(working copy)
@@ -890,6 +890,7 @@
 File "..\..\plugins\profinet\profinet.dll"
 File "..\..\plugins\sercosiii\sercosiii.dll"
 File "..\..\plugins\unistim\unistim.dll"
+File "..\..\plugins\uru\uru.dll"
 File "..\..\plugins\wimax\wimax.dll"
 File "..\..\plugins\wimaxasncp\wimaxasncp.dll"
 !include "custom_plugins.txt"
Index: Makefile.am
===================================================================
--- Makefile.am	(revision 32637)
+++ Makefile.am	(working copy)
@@ -279,6 +279,7 @@
 	-dlopen plugins/sercosiii/sercosiii.la \
 	-dlopen plugins/stats_tree/stats_tree.la \
 	-dlopen plugins/unistim/unistim.la \
+	-dlopen plugins/uru/uru.la
 	-dlopen plugins/wimax/wimax.la
 
 WTAP_PLUGIN_SOURCES = \
Index: configure.in
===================================================================
--- configure.in	(revision 32637)
+++ configure.in	(working copy)
@@ -1886,6 +1886,7 @@
   plugins/sercosiii/Makefile
   plugins/stats_tree/Makefile
   plugins/unistim/Makefile
+  plugins/uru/Makefile
   plugins/wimax/Makefile
   plugins/wimaxasncp/Makefile
   tools/Makefile
Index: plugins/Makefile.am
===================================================================
--- plugins/Makefile.am	(revision 32637)
+++ plugins/Makefile.am	(working copy)
@@ -37,6 +37,7 @@
 	sercosiii \
 	stats_tree \
 	unistim \
+	uru \
 	wimax \
 	wimaxasncp
 
